SEP-0012.md

```diff
@@ -291,44 +291,24 @@ class NDCubeABC(astropy.nddata.NDDataBase):
 
     @abc.abstractmethod
     def crop(self,
-             lower_corner: Iterable[Union[Any, None]],
-             upper_corner: Iterable[Union[Any, None]],
+             *points: Iterable[Any],
              wcs: Optional[Union[BaseHighLevelWCS, ExtraCoordsABC]] = None
              ) -> NDCubeABC:
         """
-        Crop given world coordinate objects describing the lower and upper corners of a region.
-
-        The region of interest is defined in pixel-space based on the world
-        coordinates input as the region's corners. The exact implementation
-        of the crop algorithm is not specified by this SEP.
+        Crop to the smallest cube containing the world coordinate points.
```

Comment:
To make clear earlier that this needs high level objects, while keeping the first lines short, what about doing this?
```
Crop to the smallest cube containing the world coordinate points.

World coordinates must be described by high-level coordinate objects.

Parameters
----------
```

And then something analogous for `crop_by_coords`

Reply:
I think it's probably fine, as is here. We can polish it in ndcube?

Reply:
Perhaps also this line should mention that the cropping is still done in pixel space, e.g.
`Crop to the smallest cube in pixel space containing the world coordinate points.`